ai-service/Makefile

```
@@ -1,7 +1,7 @@
 # Makefile
 
 # Combined
-.PHONY: check lint format
+.PHONY: check lint format test
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)